rocks-lib/src/build/builtin.rs

```diff
@@ -39,6 +47,7 @@ impl Build for BuiltinBuildSpec {
                     )?
                 }
                 ModuleSpec::SourcePaths(files) => {
+                    bar.set_message("Comppiling C files...");
```

Comment:
Small typo here and also in Line #61 (I didn't want to duplicate the same review comment :p)